controllers/front/OrderController.php

```diff
@@ -356,4 +358,25 @@ public function displayAjaxAddressForm()
             ),
         ]));
     }
+
+    /**
+     * Return default TOS link for checkout footer
+     *
+     * @return string
+     */
+    protected function getDefaultTermsAndConditions()
+    {
+        $cms = new CMS((int) Configuration::get('PS_CONDITIONS_CMS_ID'), $this->context->language->id);
```

Comment:
What if there is no default CMS page loaded? :)

Reply:
@PierreRambaud it's OK with my last check ? Thanks !